tools/CAT/src/app/features/news/newsExtraReducers.ts

```diff
@@ -0,0 +1,86 @@
+import { NewsDto } from '@/models/api';
+import { fetchNewsData, updateNews, addNews, deleteNews } from './newsThunks';
+import { sortNewsArrayByPinnedAndDate } from '@/helpers/sort';
+export const extraReducersConfigNews = (builder) => {
+	//get
+	builder.addCase(fetchNewsData.pending, (state) => {
+		state.loading = true;
+		state.error = null;
```

Comment:
Do we need to set every time `state.error = null` ?
That`s related to all places.